### ISSUE TITLE
Can prismaConnectionHelpers provide totalCount field?

### ISSUE THREAD

**Issue by charleslai2000** [hayes/pothos#1185]
I have tried prismaConnectionHelpers, when I connect it to a t.connection, it report error because I defined totalCount for all my connection object.

Now can just trying to resolve it from t.connection resolve function but can not find the parent node id.

**Comment by hayes** [comment]
Not currently, I that this could be added but currently I think you'd need to do it outside the helper

**Comment by charleslai2000** [comment]
Yes, I finally found a way to do it in t.connnection's resolve function.
